src/components/Network.js

```diff
@@ -184,7 +186,213 @@ class Network extends React.Component {
     return `${peerId.substr(0, 8)}...${peerId.substr(-8, 8)}`;
   }
 
-  render() {
+  renderNewUi() {
+    const loadTable = () => {
+      return (
+        <div style={{ width: '100%' }}>
+          <div className="network-data-top" style={{ marginBottom: '30px' }}>
+            <div>
+              <span>ID</span>
```

Comment:
chore: Specifically on the desktop/dark version, these titles are not being displayed with the purple color expected on Figma.

Reply:
It has been aligned and is currently corrected. Waiting Review🙂